test/CallItopServiceTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace UnitTestFiles\Test;
+
+use PHPUnit\Framework\TestCase;
+use Collector;
+
+@define('APPROOT', dirname(__FILE__, 2).'/');
+
+require_once(APPROOT.'core/collector.class.inc.php');
+
+class CallItopServiceTest extends TestCase
+{
+	/** @var \CallItopService $oCallItopService */
+	private $oCallItopService;
+
+	public function testGetCurlOptions(){
+		$aRawCurlOptions = [
+			CURLOPT_SSLVERSION => CURL_SSLVERSION_SSLv3,
+			CURLOPT_COOKIE => 'itop',
+			CURLOPT_COOKIESESSION => true
```

Comment:
Also add an option with the name as string and also the value as string like `'CURL_SSLVERSION_SSLv3'`.

Reply:
The value also needs a string constant test.

Reply:
correction made. as I moved that code in Utils class  test class/file have been renamed as well

Reply:
Test has been rewritten to highlight all usecases covered. for me the one you mentioned is tested. cf line 25 ('string key/ string value') inside UtilsTest.php.

Reply:
No, the code also supports the value to be like `'CURL_SSLVERSION_SSLv3'` and the expected result should be `3`.

Reply:
ok i got it. sorry.